src/main/java/com/adoptmeplus/enterprise/dao/CustomerDAOStub.java

```diff
@@ -21,14 +22,16 @@
 @Profile("test")
 public class CustomerDAOStub implements ICustomerDAO {
 
+    List<Customer> Customers = new ArrayList<>();
     /**
      * Fetches a list of all Customer records.
      *
      * @return A List of Customer objects representing all available customers of the specified breed.
      */
     @Override
     public List<Customer> findAll(){
-        return null;
+
```

Comment:
The stub wasn't actually passing any data, so we weren't actually testing the methods.